packages/cli/src/api/e2e.api.ts

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/naming-convention */
+import { Router } from 'express';
+import bodyParser from 'body-parser';
+import { v4 as uuid } from 'uuid';
+import config from '@/config';
+import * as Db from '@/Db';
+import { RoleNames, RoleScopes } from '@/databases/entities/Role';
+import { hashPassword } from '@/UserManagement/UserManagementHelper';
+
+if (process.env.E2E_TESTS !== 'true') {
+	console.error('E2E endpoints only allowed during E2E tests');
+	process.exit(1);
+}
+
+const tablesToTruncate = [
+	'shared_workflow',
+	'shared_credentials',
+	'webhook_entity',
+	'workflows_tags',
+	'credentials_entity',
+	'tag_entity',
+	'workflow_entity',
+	'execution_entity',
+	'settings',
+	'installed_packages',
+	'installed_nodes',
+	'user',
+	'role',
+];
+
+const truncateAll = async () => {
+	const { connection } = Db;
+	for (const table of tablesToTruncate) {
+		await connection.query(
+			`DELETE FROM ${table}; DELETE FROM sqlite_sequence WHERE name=${table};`,
+		);
+	}
```

Comment:
Minor - if you like, with `PRAGMA foreign_keys=ON` you can disable foreign keys enforcement so you can truncate in promises, and then then re-enable with `OFF`.

Reply:
I wanted to do this, but it did not work. I'm not sure why.